fit-checker/utils/change-detect/entity/Change.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Change.swift
+//  fit-checker
+//
+//  Created by Jakub Tucek on 21/01/17.
+//  Copyright © 2017 Josef Dolezal. All rights reserved.
+//
+
+import Foundation
+
+
+/// Generic change struct.
+struct Change<T> {
```

Comment:
Can we use more specific struct name here, it's closely related to classification change detection but could be easily used somewhere else by mistake..